Documentation/user-guides/running-exporters.md

```diff
@@ -96,11 +96,142 @@ By default, the `PodMonitor` and `ServiceMonitor` objects include runtime metada
 | container    | __meta_kubernetes_pod_container_name | `{name}` of the container in the scraped pod                                  |
 | endpoint     | -                                    | `{spec.Port}` or `{spec.TargetPort}` if specified                             |
 
+### Relabeling and Metric Relabeling
+
+Prometheus Operator provides the same Prometheus capability to relabel a target before scrape or a metric before sample ingestion, below you can find examples of each use case and implementation for Service or Pod monitors.
+
+#### Relabeling
+
+Relabeling is a powerful feature to dynamically rewrite the label set of a target before it gets scraped, and multiple relabeling steps can be configured per scrape configuration.
+
+> Relabel configs are applied to the label set of each target in order of their appearance in the configuration file.
+
+**Dropping label from a target**
+
+The following snippets drops the `pod` label from every metric scraped as part of the scrape job.
+
+```yaml
+- action: labeldrop
+  regex: pod
+```
+
+**Adding label to a target**
+
+The following configuration will add or replace the `team` label to all the metrics scraped as part of this job with the value `prometheus`.
+
+```yaml
+- action: replace
+  replacement: prometheus
+  targetLabel: team
+```
+
+**Filtering targets by label**
+
+The following snippet will only select targets that have the `team` label set to `prometheus` and exclude the ones that have `datacenter` set to `west_europe`.
+
+```yaml
+- sourceLabels:
+  - team
+  regex: "prometheus"
+  action: keep
+- sourceLabels:
+  - datacenter
+  regex: west_europe
+  action: drop
+```
+
+**Full example**
+
+The following snippets is using a `ServiceMonitor` to only select targets that have the `team` label set to `prometheus` and exclude the ones that have `datacenter` set to `west_europe`, but the same configuration may be used with a `PodMonitor`
```

Comment:
nit: small rephasing to avoid using the word `select`.
```suggestion
The following `ServiceMonitor` will configure Prometheus to scrape metrics from the target if they have the `team` label set to `prometheus` and `datacenter` not set to `west_europe`. The same configuration could also be used with a `PodMonitor`
```